### PR TITLE
fix(argo-rollouts): Add ConfigMap read access to support notification-engine

### PR DESCRIPTION
**Pull request by aditmeno**
With the addition of notification-engine support on Argo Rollouts, the app now needs read access to configmaps

Currently rollouts fail on master tag due to lack of configmap read access

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [x] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

**Comment by jessesuen** [comment]
> With the addition of notification-engine support on Argo Rollouts, the app now needs read access to configmaps

This is actually not in v1.0. It is coming in v1.1 so it's premature to add this RBAC

**Comment by aditmeno** [comment]
> > With the addition of notification-engine support on Argo Rollouts, the app now needs read access to configmaps
> 
> This is actually not in v1.0. It is coming in v1.1 so it's premature to add this RBAC

Should I wait until v1.1 is officially out and put this PR on the back burner till then?

The thing is, if anyone uses the master tag, these permissions are required for Argo Rollouts to operate

**Comment by jessesuen** [comment]
> The thing is, if anyone uses the master tag, these permissions are required for Argo Rollouts to operate

That's a good point. I think we can add this to make it easier to use the notification feature with master branch.

**Comment by mkilchhofer** [comment]
> Retrigger CI pipeline b1ff327
I think something is broken with the KinD version we use. Created a PR to address (and update) that: https://github.com/argoproj/argo-helm/pull/823

**Comment by mkilchhofer** [comment]
I don't use rollouts so far but is this change related to this: https://github.com/argoproj/argo-rollouts/pull/1315 ?
If yes, we should use the same manifests here I think.

**Comment by aditmeno** [comment]
> I don't use rollouts so far but is this change related to this: [argoproj/argo-rollouts#1315](https://github.com/argoproj/argo-rollouts/pull/1315) ?
> If yes, we should use the same manifests here I think.

Yes it is, I'll update it to reflect the same

File: charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml [hunk 1]
```diff
@@ -76,11 +76,13 @@ rules:
   - list
   - watch
   - patch
-# secret read access to run analysis templates which reference secrets
+# secret read access to run analysis templates which reference secrets
```
Comment (mkilchhofer) [inline charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml:79]:
```suggestion
# secret read access to run analysis templates which reference secrets
```

File: charts/argo-rollouts/templates/argo-rollouts-role.yaml [hunk 2]
```diff
@@ -66,11 +66,13 @@ rules:
   - list
   - watch
   - patch
-# secret read access to run analysis templates which reference secrets
+# secret read access to run analysis templates which reference secrets
```
Comment (mkilchhofer) [inline charts/argo-rollouts/templates/argo-rollouts-role.yaml:69]:
```suggestion
# secret read access to run analysis templates which reference secrets
```

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

**Review by mkilchhofer — APPROVED** [review]

**Pull request merged**